cost/aws/object_storage_optimization/aws_object_storage_optimization.pt

```diff
@@ -7,7 +7,7 @@ category "Cost"
 severity "low"
 default_frequency "daily"
 info(
-  version: "2.9",
+  version: "3.0",
```

Comment:
This is a minor version change so would be `2.10`
> MAJOR version when you make incompatible API changes
> MINOR version when you add functionality in a backwards compatible manner
https://semver.org/#summary